toolchain/sem_ir/formatter.cpp

```diff
@@ -772,7 +772,7 @@ class Formatter {
     out_ << InstT::Kind.ir_name();
     FormatInstructionRHS(inst);
     if (auto const_id = sem_ir_.constant_values().Get(inst_id);
-        const_id.is_constant()) {
+        const_id.is_valid() && const_id.is_constant()) {
```

Comment:
I wonder if it's worth adding an `[<invalid>]` tag or similar to the invalid case, especially if we don't expect that to happen.

Reply:
Done